components/style/gecko/wrapper.rs

```diff
@@ -1502,6 +1502,14 @@ impl<'le> TElement for GeckoElement<'le> {
             Gecko_MatchLang(self.0, override_lang_ptr, override_lang.is_some(), value.as_ptr())
         }
     }
+
+    fn is_html_document_body_element(&self) -> bool {
+        if self.get_local_name() != &*local_name!("body") {
+            return false;
+        }
+
+        unsafe { bindings::Gecko_IsBody(self.0) }
```

Comment:
Can you name this function Gecko_IsDocumentBody?  I think that would make it slightly clearer that we're checking if it's the document's (main) body element, rather than any HTML `<body>` element.